.github/workflows/auto-update-dependencies.yml

```diff
@@ -0,0 +1,32 @@
+name: Dependencies autoupdate
+
+on:
+  schedule:
+    - cron: '0 0 * * *' # run daily at 12:00 am
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  auto-update-dependencies:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout the head commit of the branch
+      uses: actions/checkout@v2
+      with:
+        persist-credentials: false 
+        
+    - name: Install Linux requirements
+      run: |
+        apt_dependencies="git curl libssl-dev pkg-config libudev-dev libv4l-dev"
+        echo "Run apt update and apt install the following dependencies: $apt_dependencies"
+        sudo apt update
+        sudo apt install -y $apt_dependencies
+    
+    - name: Check for dependency updates
+      uses: romoh/dependencies-autoupdate@v1.1
```

Comment:
could we have just used `dependencies-autoupdate@v1.1` since you [published the action](https://github.com/marketplace/actions/autoupdate-dependencies)?

Reply:
The action need to have the original owner/repo path. Some creates an organization with friendly names to publish their actions (ie: actions-rs)